tests/www/test_views.py

```diff
@@ -254,6 +254,66 @@ def test_prefill_form_null_extra(self):
         cmv.prefill_form(form=mock_form, pk=1)
 
 
+class TestCopyConnectionModelView(TestBase):
+    def setUp(self):
+        super().setUp()
+        conn1 = Connection(
+            conn_id='aws_mongo',
+            conn_type='FTP',
+            description='MongoDB',
+            host='192.168.55.11',
+            schema='airflow',
+            port='5566',
+        )
+
+        # Already copied and modified
+        conn2 = Connection(
+            conn_id='aws_mongo__1',
+            conn_type='FTP',
+            description='MongoDB2',
+            host='192.168.55.12',
+            schema='airflow',
+            port='5567',
+        )
+
+        conn3 = Connection(
+            conn_id='aws_mysql',
+            conn_type='FTP',
+            description='MongoDB2',
+            host='192.168.55.12',
+            schema='airflow',
+            port='5567',
+        )
+
+        self.clear_table(Connection)
+        self.session.add_all([conn1, conn2, conn3])
+        self.session.commit()
+
+    def tearDown(self):
+        self.clear_table(Connection)
+        super().tearDown()
+
+    def test_copy_connection(self):
+        """ Test copy multiple connection with suffix(already copied )"""
+
+        mock_form = mock.Mock()
+        mock_form.data = {"action": "mulcopy", "rowid": [1, 2, 3]}
+        resp = self.client.post('/connection/action_post', data=mock_form.data, follow_redirects=True)
+
+        expected_result = {
+            'aws_mongo',
+            'aws_mongo__1',
+            'aws_mongo__2',
+            'aws_mongo__3',
```

Comment:
Is it fair to treat all `aws_mongo` variants as essentially interchangeable in terms of numbering them? Would `aws_mongo__1__1` make more sense than `aws_mongo__3` to make clear that it's a copy of `aws_mongo__1`? It doesn't look as nice, but it seems like people would copy `aws_mongo__1` because they want a copy of that exact variant.

Reply:
Here in test case i have used '__1' as existing connection to test copy connection which were already copied connection and not renamed after copy.

For Pattern matching in RegEx we have used '__/d+$' to differentiate system(airflow) assigned suffix (we have used same Regex for Task and TaskGroup decorators). User can use  single underscore '_' or '-' to assign number to connection name which will make it start from 1.

For Example 

'mongo_1' -> 'mongo_1__1'
'mongo-1' -> 'mongo-1__1'